pkg/process/events/model/model_sysprobe.go

```diff
@@ -43,3 +43,80 @@ func ProcessMonitoringtoProcessEvent(e *ProcessMonitoringEvent) *ProcessEvent {
 		ExitTime:       e.ExitTime,
 	}
 }
+
+// ProcessEventToProcessMonitorinEvent converts a ProcessEvent to a ProcessMonitoringEvent
+// It's used during tests to mock a ProcessMonitoringEvent message
+func ProcessEventToProcessMonitorinEvent(e *ProcessEvent) *ProcessMonitoringEvent {
```

Comment:
nit: typo? `ProcessEventToProcessMonitorinEvent` -> `ProcessEventToProcessMonitoringEvent`

Reply:
Fixed in https://github.com/DataDog/datadog-agent/commit/3483bcc299ebda145257a11d10995abdb11148af.
Thanks for spotting it!